### PR TITLE
[CAT-806] adding github pr script

### PR DESCRIPTION
**Pull request by goatrocks**
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] Jira story/task is put into PR title
- [ ] Code has been self-reviewed
- [ ] Hard-to-understand areas have been commented
- [ ] Documentation has been updated
- [ ] My changes generate no new warnings
- [ ] Unit tests have been added for base functionality and known edge cases
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by goatrocks** [comment]
start of a script for testing with harness -- we should pass in $TAG and $GITHUB_TOKEN through harness

File: scripts/push_docs.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+git add test.md
+git commit -m "a set of doc changes"
+git push --set-upstream origin docs-version-$TAG
+curl -L \
+  -X POST \
+  -H "Accept: application/vnd.github+json" \
+  -H "Authorization: Bearer $GITHUB_TOKEN" \
+  -H "X-GitHub-Api-Version: 2022-11-28" \
+  https://api.github.com/repos/IndicoDataSolutions/indico-readme/pulls \
+  -d '{"title":"Amazing new feature","body":"Please pull these awesome changes in!","head":"docs-version-'"$TAG"'","base":"main"}'
```
Comment (meghanhickey) [inline scripts/push_docs.sh:16]: Harness allows you to [reference codebase variables](https://developer.harness.io/docs/continuous-integration/use-ci/codebase-configuration/built-in-cie-codebase-variables-reference/#reference-codebase-variables), like the PR title and body, and pass them as environment variables. How about, when making a PR into the docs repo, we use the same title as the PR into the SDK repo?

Comment (goatrocks) [reply scripts/push_docs.sh:16]: sounds perfect!

**Comment by meghanhickey** [comment]
I think it might make more sense to put this script in the indico-readme repo, since it seems to be language agnostic :thinking: what do you think?

**Comment by meghanhickey** [comment]
> I think it might make more sense to put this script in the indico-readme repo, since it seems to be language agnostic 🤔 what do you think?

I just realized that without this script, the python repo has no way to access the indico-readme repo :sweat_smile: so maybe ignore me

**Review by meghanhickey — APPROVED** [review]
